src/actions/images.ts

```
@@ -205,7 +205,7 @@ export const beat_graph_data = {
       defaultValue: {},
     },
     audioChecker: {
-      agent: async (namedInputs: { movieFile: string; imageFile: string; soundEffectFile: string }) => {
+      agent: async (namedInputs: { movieFile: string; imageFile: string; soundEffectFile: string, index: number }) => {
```

ESLint

[error] Replace `,` with `;`

(prettier/prettier)